Marlin/Configuration.h

```diff
@@ -663,6 +663,9 @@
  * readings with inductive probes and piezo sensors.
  */
 //#define PROBING_HEATERS_OFF       // Turn heaters off when probing
+#if ENABLED(PROBING_HEATERS_OFF)
+	//#define WAIT_FOR_BED_HEATER     //Wait for bed to heat back up between probes (can improve accuracy)
```

Comment:
Tabs are verboten. See if you can set your `vim` / `notepad++` / etc. settings to use two spaces instead of tabs.

Reply:
Ah will do.